### PR TITLE
add doc for cloudwatch container insights

### PR DESCRIPTION
**Pull request by pxaws**
Add public doc for supporting CloudWatch Container Insights in EKS. This doc is a part of the next release for AWS OpenTelemetry Collector.

### CODE REVIEW

File: src/docs/getting-started/cloudwatch-container-insights.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import { Link } from "gatsby"
+import SectionSeparator from "components/MdxSectionSeparator/sectionSeparator.jsx"
+import imgLogGroup from "assets/img/docs/gettingStarted/containerInsights/log-group.png"
+import imgPodMetrics from "assets/img/docs/gettingStarted/containerInsights/pod-metrics.png"
+
+[CloudWatch Container Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html) collect, aggregate, 
+and summarize metrics and logs from your containerized applications and microservices. Data are collected as as performance log events 
+using [embedded metric format](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format.html). 
+These performance log events are entries that use a structured JSON schema that enables high-cardinality data to be ingested and stored at scale. 
+From this data, CloudWatch creates aggregated metrics at the cluster, node, pod, task, and service level as CloudWatch metrics.
```
Comment (mxiamxia) [inline src/docs/getting-started/cloudwatch-container-insights.mdx:18]:
```suggestion
Amazon CloudWatch can creates the aggregated CloudWatch metrics for the cluster, node, pod, task, and service level from the received EMF data.
```

File: src/docs/getting-started/cloudwatch-container-insights.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+import imgLogGroup from "assets/img/docs/gettingStarted/containerInsights/log-group.png"
+import imgPodMetrics from "assets/img/docs/gettingStarted/containerInsights/pod-metrics.png"
+
+[CloudWatch Container Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html) collect, aggregate, 
+and summarize metrics and logs from your containerized applications and microservices. Data are collected as as performance log events 
+using [embedded metric format](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format.html). 
+These performance log events are entries that use a structured JSON schema that enables high-cardinality data to be ingested and stored at scale. 
+From this data, CloudWatch creates aggregated metrics at the cluster, node, pod, task, and service level as CloudWatch metrics. 
+
+Exsting CloudWatch Container Insights relies on ECS Agent and CloudWatch Agent to collect infrastructure metrics for many resources such as
```
Comment (mxiamxia) [inline src/docs/getting-started/cloudwatch-container-insights.mdx:20]:
```suggestion
CloudWatch Container Insights has been supported by ECS Agent and CloudWatch Agent to collect infrastructure metrics for many resources such as
```

File: src/docs/getting-started/cloudwatch-container-insights.mdx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  * Kubernetes platforms on Amazon EC2
+
+For Amazon ECS, the cluster and service level metrics are already supported ([see the public AWS doc for details](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy-container-insights-ECS-adot.html))
+and the support for instance level metrics is working in progress. For Amazon EKS and Kubernetes platforms on Amazon on EC2, all the infrastructure metrics are supported. 
+In this tutorial, we will walk through how to enable CloudWatch Container Insights with AWS OTel Collector for an EKS cluster. 
+
+<SectionSeparator />
+
+## Getting Started
+To use AWS OTel Collector to collect infrastructure metrics for your cluster, we need to [make sure all the prerequisites are satisfied.](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html)
```
Comment (mxiamxia) [inline src/docs/getting-started/cloudwatch-container-insights.mdx:34]:
```suggestion
To use AWS OTel Collector to collect infrastructure metrics for your service cluster, we need to make sure [all the prerequisites](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html) are satisfied.
```

File: src/docs/getting-started/cloudwatch-container-insights.mdx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+under the log group `/aws/containerinsights/ci-demo/performance` and each corresponds to one cluster node:
+<img src={imgLogGroup} alt="Diagram" style="margin: 30px 0;" />
+
+The metrics that Container Insights collects are also available in CloudWatch automatic dashboards as different resource types, such as 
+`EKS Clusters`, `EKS Namespaces`, `EKS Nodes`, `EKS Services`, and `EKS Pods`. Here is a screenshot for the pod level metrics for 
+a cluster named `ci-demo`:
+<img src={imgPodMetrics} alt="Diagram" style="margin: 30px 0;" /> 
+
+
+## Configurations to support CloudWatch Container Insights for EKS
```
Comment (mxiamxia) [inline src/docs/getting-started/cloudwatch-container-insights.mdx:59]:
```suggestion
## Configurations to support CloudWatch Container Insights for EKS EC2
```

File: src/docs/getting-started/cloudwatch-container-insights.mdx [hunk 5]
```diff
@@ -0,0 +1,164 @@
+---
+title: 'Using CloudWatch Metrics with AWS Distro for OpenTelemetry'
+description:
+    CloudWatch Container Insights collects, aggregates, and summarize metrics and logs from your containerized applications and microservices. 
+    In this tutorial, we will walk through how to enable CloudWatch Container Insights with AWS OTel Collector for an EKS cluster.
```
[the remark this answers is not in the document]
Comment (pxaws) [reply src/docs/getting-started/cloudwatch-container-insights.mdx:5]: Will change accordingly.

File: src/docs/getting-started/cloudwatch-container-insights.mdx (hunk 1 above)
Comment (pxaws) [reply src/docs/getting-started/cloudwatch-container-insights.mdx:18]: Will change accordingly.

File: src/docs/getting-started/cloudwatch-container-insights.mdx (hunk 2 above)
Comment (pxaws) [reply src/docs/getting-started/cloudwatch-container-insights.mdx:20]: Will change accordingly.

File: src/docs/getting-started/cloudwatch-container-insights.mdx (hunk 4 above)
Comment (pxaws) [reply src/docs/getting-started/cloudwatch-container-insights.mdx:59]: Will change accordingly.

File: src/docs/getting-started/cloudwatch-container-insights.mdx (hunk 3 above)
Comment (pxaws) [reply src/docs/getting-started/cloudwatch-container-insights.mdx:34]: Will change accordingly.

**Review by mxiamxia — APPROVED** [review]
LGTM

**Review by alolita — COMMENTED** [review]
On hold until PRs are merged in OTEL.

**Comment by pingleig** [comment]
Preview https://trusting-lewin-3415ee.netlify.app/docs/getting-started/container-insights/eks-infra

File: src/docs/getting-started/container-insights/eks-infra.mdx [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+In this tutorial, we will walk through how to enable CloudWatch Container Insights infrastructure metrics with AWS OTel Collector for an EKS EC2 cluster. 
+
+<SectionSeparator />
+
+## Getting Started
+To use the AWS OTel Collector to collect infrastructure metrics for a service cluster, we need to make sure [all the prerequisites](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html) are satisfied.
+
+Then we can deploy the AWS OTel Collector as a daemon set to the cluster by entering the following command:
+```
+curl https://raw.githubusercontent.com/aws-observability/aws-otel-collector/main/deployment-template/eks/otel-container-insight-infra.yaml |
```
Comment (mxiamxia) [inline src/docs/getting-started/container-insights/eks-infra.mdx:22]:
```suggestion
curl https://raw.githubusercontent.com/aws-observability/aws-otel-collector/main/deployment-template/eks/otel-container-insights-infra.yaml |
```

Comment (pxaws) [reply src/docs/getting-started/container-insights/eks-infra.mdx]: Thanks for catching it!

**Review by alolita — CHANGES_REQUESTED** [review]
Please correct all usage of:
* Container Insight Prometheus - trademark violation. please fix - see comments on related PR
* AWS OTel Collector to ADOT Collector
* collector to Collector

**Comment by pxaws** [comment]
> Please correct all usage of:
> 
> * Container Insight Prometheus - trademark violation. please fix - see comments on related PR
> * AWS OTel Collector to ADOT Collector
> * collector to Collector

They are addressed in the latest commit.

**Review by mxiamxia — APPROVED** [review]

**Review by alolita — APPROVED** [review]

**Pull request merged**